docs/reference/search/count.asciidoc

```diff
@@ -1,11 +1,91 @@
 [[search-count]]
 === Count API
 
-The count API allows to easily execute a query and get the number of
-matches for that query. It can be executed across one or more indices.
-The query can either be provided using a simple query string as a
-parameter, or using the <<query-dsl,Query DSL>> defined within the request
-body. Here is an example:
+Gets the number of matches for a search query.
+
+[source,js]
+--------------------------------------------------
+GET /twitter/_count?q=user:kimchy
+--------------------------------------------------
+//CONSOLE
```

Comment:
```suggestion
// TEST[setup:twitter]
```

This is likely causing the CI to fail.

`//CONSOLE`  is no longer needed with `[source,console].